common/utils.js

```diff
@@ -443,7 +443,7 @@ const calculateDomainSeparator = (chain, router, network) => keccak256(Buffer.fr
 const getItsEdgeContract = (chainConfig) => {
     const itsEdgeContract =
         chainConfig.contracts.InterchainTokenService?.address ||
-        chainConfig.contracts.InterchainTokenService?.objects?.ChannelId ||
+        chainConfig.contracts.ITS?.objects?.ChannelId ||
```

Comment:
This has been renamed to `InterchainTokenService` (devnet is using old version). Support both names for now with a TODO to remove the old name once devnet is redeployed